### PR TITLE
Fix-repo-to-packetloop

### PR DESCRIPTION
**Pull request by lenfree**
Move repo from personal to packetloop org so anyone within the org
have full control to add/fix/merge any code changes

@comozo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,22 @@ ws-healthcheck
 
 A simple executable to quickly make a websocket healthcheck.
 
+[![Build Status](https://travis-ci.org/packetloop/ws-healthcheck.svg?branch=master)](https://travis-ci.org/packetloop/ws-healthcheck.svg?branch=master)
```
Comment (comozo) [inline README.md]: I think it is better to have it build with circle-ci. It will be more consistent with the rest of the platform. :-)

Comment (lenfree) [inline README.md]: This is because I have yet to find a way how to do releases in circleci. 😸  However, I will find time to port this back to circleci.

Comment (comozo) [inline README.md]: ok

**Comment by comozo** [comment]
👍

**Pull request merged**